README.md

```diff
@@ -12,81 +12,79 @@
 
 ## Table of Contents
 
-- [Get started for free](#get-started-for-free)
-- [Docs](#docs)
+- [Getting started with PostHog Cloud](#getting-started-with-posthog-cloud-recommended)
+- [Self-hosting the open-source hobby deploy](#self-hosting-the-open-source-hobby-deploy-advanced)
+- [Setting up PostHog](#setting-up-posthog)
+- [Learning more about PostHog](#learning-more-about-posthog)
 - [Contributing](#contributing)
-- [Philosophy](#philosophy)
 - [Open-source vs paid](#open-source-vs-paid)
 
-## Get started for free
+## Getting started with PostHog Cloud (Recommended)
 
-### PostHog Cloud (Recommended)
+The fastest and most reliable way to get started with PostHog is signing up for free to [PostHog Cloud](https://us.posthog.com/signup) or [PostHog Cloud EU](https://eu.posthog.com/signup). Your first 1 million events, 5k recordings, 1M flag requests, 100k exceptions, and 250 survey responses are free every month, after which you pay based on usage.
 
-The fastest and most reliable way to get started with PostHog is signing up for free to [PostHog Cloud](https://us.posthog.com/signup) or [PostHog Cloud EU](https://eu.posthog.com/signup). Your first 1 million events (and 5k replays) are free every month, after which you pay based on usage.
+## Self-hosting the open-source hobby deploy (Advanced)
 
-### Open-source hobby deploy (Advanced)
+If you want to self-host PostHog, you can deploy a hobby instance in one line on Linux with Docker (recommended 4GB memory):
 
-You can deploy a hobby instance in one line on Linux with Docker (recommended 4GB memory):
+```bash 
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/posthog/posthog/HEAD/bin/deploy-hobby)" 
+```
 
- ```bash 
-  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/posthog/posthog/HEAD/bin/deploy-hobby)" 
- ``` 
+Open source deployments should scale to approximately 100k events per month, after which we recommend [migrating to a PostHog Cloud](https://posthog.com/docs/migrate/migrate-to-cloud). 
 
-Open source deployments should scale to approximately 100k events per month, after which we recommend migrating to a PostHog Cloud instance. See our [docs for more info and limitations](https://posthog.com/docs/self-host/open-source/deployment). Please note that we do not provide customer support for open source deployments. 
+We *do not* provide customer support or offer guarantees for open source deployments. See our [self-hosting docs](https://posthog.com/docs/self-host), [troubleshooting guide](https://posthog.com/docs/self-host/deploy/troubleshooting), and [disclaimer](https://posthog.com/docs/self-host/open-source/disclaimer) for more info.
 
-## Docs
-![ui-demo](https://user-images.githubusercontent.com/85295485/144591577-fe97e4a5-5631-4a60-a684-45caf421507f.gif)
```

Comment:
It was nice that we had a gif, maybe we could add something new here? Or do you believe the video demo is enough?

Reply:
The problem is that gifs are huge files and videos don't autoplay. There's also less of an obvious spot to put it and products to show off. I think we should rely on the demo video and people actually going to the site, but open to chnaging.